store/migration/10001__init_schema.sql

```diff
@@ -277,6 +277,10 @@ CREATE TABLE project_member (
     updater_id INTEGER NOT NULL REFERENCES principal (id),
     updated_ts BIGINT NOT NULL DEFAULT (strftime('%s', 'now')),
     project_id INTEGER NOT NULL REFERENCES project (id),
+    -- allowed role_provider are 'BYTEBASE', 'GITLAB_SELF_HOST'.
+    role_provider TEXT NOT NULL DEFAULT "BYTEBASE",
+    -- role_provider_payload is determined by the type of role_provider
+    role_provider_payload TEXT NOT NULL DEFAULT,
```

Comment:
```suggestion
    -- payload is determined by the type of role_provider
    payload TEXT NOT NULL DEFAULT,
```

Reply:
nit: I would also put role_provider and payload at last (after principal_id).

payload contains may data. So a SELECT * FROM project_members displays better output if we put scalar value first.